utils/docker/run-doc-update.sh

```diff
@@ -74,8 +74,7 @@ git clean -dfx
 echo "Copy content"
 rsync -a ${ARTIFACTS_DIR}/web_linux/ ./content/pmdk/manpages/linux/${TARGET_DOCS_DIR}/ --delete
 rsync -a ${ARTIFACTS_DIR}/web_windows/ ./content/pmdk/manpages/windows/${TARGET_DOCS_DIR}/ --delete \
-	--exclude='librpmem'	\
```

Comment:
It looks like the documentation for librpmem is already broken. On pmem.io you can see that documentation for functions for older releases (where librpma was available) is missing. I think that our automation system was not designed to remove functions from the API. As long as we were only adding features there was no such problem. Those scripts that you modify now always run during documentation updates and should have backward compatibility. I suggest you investigate this issue.

Reply:
Please undo this change until the fix is figured out, without this change I will be able to merge this PR.

Reply:
Done